### PR TITLE
Updated aws instance types actual for 2019.02.10

### PR DESCRIPTION
**Pull request by miry**
Sync instance type for aws via `make generate`

### CODE REVIEW

File: cluster-autoscaler/cloudprovider/aws/ec2_instance_types.go [hunk 1]
```diff
@@ -705,6 +783,18 @@ var InstanceTypes = map[string]*instanceType{
 		MemoryMb:     249856,
 		GPU:          4,
 	},
+	"p3dn": {
```
Comment (Jeffwan) [inline cluster-autoscaler/cloudprovider/aws/ec2_instance_types.go]: em.. It looks weird. Not sure why here's a separate p3dn instance type here. 
The other thing is `p3dn.24xlarge` is not fully tested on kubernetes. (it supposed to be release together with latest CNI, and this instance type also need whitelist)

Let me double confirm and then come back to you.

Comment (Jeffwan) [reply cluster-autoscaler/cloudprovider/aws/ec2_instance_types.go]: Looks lik p3dn belongs to dedicated host type. Not an issue.

**Comment by mwielgus** [comment]
@Jeffwan  - does it mean LGTM?

File: cluster-autoscaler/cloudprovider/aws/ec2_instance_types.go (hunk 1 above)
Comment (miry) [reply cluster-autoscaler/cloudprovider/aws/ec2_instance_types.go]: @Jeffwan should I do something more? Should I check makefile to ignore p3dn?

Comment (Jeffwan) [reply cluster-autoscaler/cloudprovider/aws/ec2_instance_types.go]: Just confirmed that p3dn.24xlarge test is done. Right now, AFAIK no one use dedicated host, I think we can still use API results directly. Nothing needs to be changed.

**Comment by Jeffwan** [comment]
> @Jeffwan - does it mean LGTM?

Yes. Confirmed and LGTM

**Pull request merged**